tests/RPCClient.test.ts

```diff
@@ -1285,27 +1205,22 @@ describe(`${RPCClient.name}`, () => {
           idGen,
           timeoutTime,
         });
-
         await rpcClient.duplexStreamCaller<
           JSONRPCRequestParams,
           JSONRPCResponseResult
         >(methodName, {
           timer: Infinity,
         });
-
         const ctx = await ctxP;
         expect(ctx.timer.delay).toBe(Infinity);
         ctx.timer.cancel();
         await ctx.timer.catch(() => {});
       },
     );
-    test.prop(
-      {
-        messages: specificMessageArb,
-      },
-      { numRuns: 1 },
-    )(
-      'Check that ctx is provided to the middleware and that the middleware can reset the timer',
+    // Given the message, I feel that the test should be split into two separate
+    // tests.
+    test.prop({ messages: specificMessageArb }, { numRuns: 1 })(
+      'ctx should be provided to the middleware and the middleware should be able to reset the timer',
```

Comment:
The `and` word in the test description makes me feel that the test should be split into two separate tests instead of a single combined test, but I'm not too familiar with the code, so I didn't make this change yet.

Reply:
Just remove the comment. It seems to be no longer relevant.